packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -359,6 +359,21 @@ interface DatabaseClusterBaseProps {
    * @default - IPV4
    */
   readonly networkType?: NetworkType;
+
+  /**
+   * The Active Directory directory ID, required for setting up Kerberos authentication, to create the DB cluster in.
+   *
+   * @default - Do not join domain
+   */
+  readonly domain?: string;
+
+  /**
+   * The IAM role to be used when making API calls to the Directory Service. The role needs the AWS-managed policy
+   * AmazonRDSDirectoryServiceAccess or equivalent.
+   *
+   * @default - The role will be created for you if `DatabaseClusterBaseProps#domain` is specified
```

Comment:
Would be nice to know what permissions are granted with the default role. Also is `DatabaseClusterBaseProps#domain` supposed to be `DatabaseClusterBaseProps.domain`?